coderd/users.go

```diff
@@ -867,9 +876,52 @@ func (api *API) putUserStatus(status database.UserStatus) func(rw http.ResponseW
 			})
 			return
 		}
+		httpapi.Write(ctx, rw, http.StatusOK, db2sdk.User(targetUser, organizations))
+	}
+}
+
+func (api *API) notifyUserStatusChanged(ctx context.Context, user database.User, status database.UserStatus) error {
+	var key string
+	var adminTemplateID, ownerTemplateID uuid.UUID
+	switch status {
+	case database.UserStatusSuspended:
+		key = "suspended_account_name"
+		adminTemplateID = notifications.TemplateUserAccountSuspended
+		ownerTemplateID = notifications.TemplateYourAccountSuspended
```

Comment:
Nit: `owner` is a bit of a strange name; perhaps `userTemplateID` or `personalTemplateID`?